app/src/components/Landing.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import Contributors from './Contributors';
+import { Link } from 'react-router-dom'
+
+import logoAustinMonitor from '../images/landing/austin-monitor-circle-logo.png';
+import logoGlasshouse from '../images/landing/glasshouse-policy-circle-logo.png';
+import logoMozilla from '../images/landing/mozilla-circle-logo.png';
+import logoOpenAustin from '../images/landing/open-austin-circle-logo.png';
+import IconVoice from '../images/landing/voice-icon.svg';
+import Confetti from '../images/landing/confetti.svg';
+import HomepageScreens from '../images/landing/homepage-screens.svg';
+
+export default class Landing extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { contributors: [] };
+  }
+
+  componentWillMount() {
+    fetch(this.props.contributorsURL)
```

Comment:
Do we need any kind of polyfill to get fetch to work across all browsers?
https://caniuse.com/#search=fetch